data/bedrock/1.16.201/attributes.json

```diff
@@ -15,7 +15,7 @@
   },
   {
     "name": "movementSpeed",
-    "resource": "generic.movement_speed",
+    "resource": "minecraft:movement",
```

Comment:
is this actually only incorrect attribute ? doesn't seem to align with other resourceIds